efro/.htaccess

```diff
@@ -7,20 +7,23 @@ Options -MultiViews
 # Directive to ensure *.rdf files served as appropriate content type,
 # if not present in main apache config
 AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
 
-# Rewrite engine
-RewriteEngine on
-RewriteBase /efro/
-
-# Rewrite rule to serve HTML content from the vocabulary URI
-RewriteCond %{HTTP_ACCEPT} !application/rdf+xml.*(text/html|application/xhtml+xml)
-RewriteCond %{HTTP_ACCEPT} text/html [OR]
-RewriteCond %{HTTP_ACCEPT} application/xhtml+xml [OR]
-RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^efro$ https://rgu-computing.github.io/EFRO/EFRO/docs/index-en.html [R=303,L]
-
-# Default response - owl rdf
-RewriteRule ^efro$ https://rgu-computing.github.io/EFRO/EFRO/EFRO.rdf [R=303,NE,L]
+<IfModule mod_rewrite.c>
```

Comment:
Some comments:
- The IfModule isn't needed on this site.  mod_rewrite will always be on.
- The RewriteBase is only used for local files, and this service only uses redirects.
- This is mapping URLs like https://w3id.org/efro/efro.  Rules are relative to the current directory.  Did you mean to have the duplication?  I'm guessing so since the target also does that.